src/app/@client/(content)/home/page.tsx

```diff
@@ -1,15 +1,35 @@
 import BlogShowcase from '@/components/Blogs/BlogShowcase';
 import React from 'react';
 import SimpleCTA from '@/components/CTA/SimpleCTA';
+import ClientScheduleAppointment from '@/components/Calendars/ClientScheduleAppointment';
+import { serverFetcher } from '@/lib/fetchers/serverFetcher';
+import { StrapiResponse } from '@/types/StrapiResponse';
+import { Availability } from '@/types/Availability';
+
+const fetchAvailabilities = async () => {
+    const today = new Date().toISOString().substring(0, 10);
```

Comment:
I just noticed this, why are we doing the substring again? Can you try without even putting this to a string to see if returns you the correct result